README.md

```diff
@@ -5,20 +5,30 @@
 [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/awslabs/karpenter/issues)
 
 ![](website/static/banner.png)
+{{% alert title="Note" color="primary" %}}
```

Comment:
I didn't know GitHub markdown supported this!

Reply:
I don't know if it does. Not rendering for me

Reply:
Oh darn. That's a Hugo/docsy thing. I forgot I'm not in the docs section. I'll fix it.